src/components/input/input.tsx

```diff
@@ -325,6 +325,13 @@ export class Input
   /** The component's value. */
   @Prop({ mutable: true }) value = "";
 
+  /**
+   * Contains the component's list of files when `type` is `"file"`.
```

Comment:
@geospatialem can I get a quick doc review?

Reply:
On it :)

Reply:
WDYT about...

> When `type` is `"file"`, specifies the component's selected files.